test/cli-test.js

```diff
@@ -7,13 +7,31 @@ var cliTxt = path.resolve(_files, 'cli.txt');
 var cliPartialsTxt = path.resolve(_files, 'cli_with_partials.txt');
 var moduleVersion = require('../package').version;
 
-var exec = require('child_process').exec;
+var child_process = require('child_process');
```

Comment:
Yeah, I was wondering why the `require('child_process').exec` was separate originally